src/controller/tests/data_model/TestRead.cpp

```diff
@@ -173,31 +182,31 @@ void TestReadInteraction::TestReadAttributeResponse(nlTestSuite * apSuite, void
 
     // Passing of stack variables by reference is only safe because of synchronous completion of the interaction. Otherwise, it's
     // not safe to do so.
-    auto onSuccessCb = [apSuite, &onSuccessCbInvoked](const app::ConcreteAttributePath & attributePath, const auto & dataResponse) {
+    auto onSuccessCb = [apSuite, &onSuccessCbInvoked](const app::ConcreteDataAttributePath & attributePath,
+                                                      const auto & dataResponse) {
         uint8_t i = 0;
-
+        NL_TEST_ASSERT(apSuite, attributePath.mDataVersion == kDataVersion);
         auto iter = dataResponse.begin();
         while (iter.Next())
         {
             auto & item = iter.GetValue();
             NL_TEST_ASSERT(apSuite, item.fabricIndex == i);
             i++;
         }
-
         NL_TEST_ASSERT(apSuite, i == 4);
         NL_TEST_ASSERT(apSuite, iter.GetStatus() == CHIP_NO_ERROR);
-
         onSuccessCbInvoked = true;
     };
 
     // Passing of stack variables by reference is only safe because of synchronous completion of the interaction. Otherwise, it's
     // not safe to do so.
-    auto onFailureCb = [&onFailureCbInvoked](const app::ConcreteAttributePath * attributePath, CHIP_ERROR aError) {
+    auto onFailureCb = [&onFailureCbInvoked](const app::ConcreteDataAttributePath * attributePath, CHIP_ERROR aError) {
         onFailureCbInvoked = true;
     };
 
+    chip::Optional<chip::DataVersion> dataVersion;
```

Comment:
This whole file is `using namespace chip;`.  Why all the `chip::` bits?  Please take them out, from all the changes made to this file.

Reply:
Updated for this file, done, thanks